htdocs/comm/propal/list.php

```diff
@@ -471,6 +471,7 @@
 	// Lignes des champs de filtre
 	print '<form method="POST" id="searchFormList" action="'.$_SERVER["PHP_SELF"].'">';
 	if ($optioncss != '') print '<input type="hidden" name="optioncss" value="'.$optioncss.'">';
+	if ($sall) print '<input type="hidden" name="sall" value="'.$sall.'" />';
```

Comment:
If you add sall here, it means you keep the initial search all the time.
If you then make another filter with another criteria, the result will match all criteria and not the new one.  This may user thinking there is a bug.
Which use case/bug does this ifx ?

Reply:
i think you are right; i close this PR